### DISCUSSION TITLE
egg for game Tachyon

### DISCUSSION THREAD

**Discussion by Alfatossa** [pterodactyl/panel#3067] [Feature Requests]
There is a game called Tachyon that has support for Linux and that would be great to be able to move it to pterodactyl using the files since windows tend be a bitch and requires to turn of the firewall to let other people to join

**Comment by parkervcp** [comment]
I take egg requests in my repo here - https://github.com/parkervcp/eggs

**Reply by Alfatossa** [reply]
so i cant just copy and paste it in there?

**Reply by parkervcp** [reply]
in theory yes you can but you will probably need to do some work.

**Reply by LoneWolfGmer** [reply]
> I take egg requests in my repo here - https://github.com/parkervcp/eggs

can you update the egg for tachyons?
it's a bit outdated running from the community version of the game and now we got like 
Tachyons a0831 which as you know is quite newer than the community version being V821

**Comment by erwingame** [comment]
I'm interested.

**Reply by Alfatossa** [reply]
i got it working so here is a link to the json i have for it

https://www.ultimacraftmc.com/dwnld/egg-tachyon.json

you just have to extract all files from the server folder to the main folder otherwise it wont be able to start

**Comment by LoneWolfGmer** [comment]
> There is a game called Tachyon that has support for Linux and that would be great to be able to move it to pterodactyl using the files since windows tend be a bitch and requires to turn of the firewall to let other people to join

I lack said issue personally. might of been a feature on the now outdated feature of the game. which is now upddated
